normandy/recipes/api/serializers.py

```diff
@@ -37,6 +38,25 @@ class Meta:
         ]
 
 
-class BundleSerializer(serializers.Serializer):
+class BundleSerializer(serializers.ModelSerializer):
```

Comment:
I don't think you want to do this, since `Bundle` isn't a Django model. Was something not working with this?